inference/worker/utils.py

```diff
@@ -76,6 +76,7 @@ def wait_for_inference_server(inference_server_url: str, timeout: int = 600):
 
 
 def text_to_events(text: str, seed: int | None = None, pause: float = 0.0):
+    time.sleep(2)
```

Comment:
did not understand how this helps in computing message queue positions. is this needed ?